lib/formats/sh.c.in

```diff
@@ -269,23 +269,23 @@ static void rm_fmt_head(RmSession *session, RmFmtHandler *parent, FILE *out) {
 
     self->line_count = 0;
     self->session = session;
-    self->user_cmd = rm_fmt_get_config_value(session->formats, "sh", "cmd");
+    self->user_cmd = rm_fmt_get_config_value(session->cfg->formats, "sh", "cmd");
```

Comment:
Actually I'd prefer to make it a ~global~ GLOBAL in `formats.c` and not expose the `RmFmtTable` structure in `formats.h`.  Then this would become:
`self->user_cmd = rm_fmt_get_config_value("sh", "cmd");`

Yes "globals are evil" but currently `session` (and hence `session->formats`), while note technically global, is accessible to just about every procedure (for example every RmFile has file->session).  So we get the main disadvantage of a global (unprotected data that can be modified by any function) plus the disadvantage of non-globals (having to pass parameters around).